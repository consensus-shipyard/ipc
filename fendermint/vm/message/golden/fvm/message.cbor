8a1b33884b011fd74a91583103ff5d465bc44b017bef29964bc0bc6cc84b7207033c03285e1872ad006b3da38da39223ff72b3f74902168ed748a60782583604adb5ece58cd986a41dcbf731d6b59f3496aac962e042c98bdd6b69ab279a38667d64c19bd8b9db609764980c133d0186eb1a01bc6e1b8afe2f3af5d4b9684900f909674792abf6791b9a59f60bc759fdb65821004636280b3a37fec660739b79da958f5ddfad510da03684c22f93a50c9038b23458190022e46edbc2d0ced0aeb9589fc8097adeae29fe81ddc42c3f1ba23fe4736808338944ac8ff874