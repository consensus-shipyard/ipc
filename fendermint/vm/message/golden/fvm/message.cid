0171a0e40220533f119b5d3ee07e1f4a7b3f21a0b5c28ee2a87cc554a2857adbcf37f2ebfbd2