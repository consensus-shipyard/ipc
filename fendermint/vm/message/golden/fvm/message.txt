Message { version: 3713300360968227473, from: Address { payload: Delegated(DelegatedAddress { namespace: 2109965875153672877, length: 44, buffer: [203, 247, 49, 214, 181, 159, 52, 150, 170, 201, 98, 224, 66, 201, 139, 221, 107, 105, 171, 39, 154, 56, 102, 125, 100, 193, 155, 216, 185, 219, 96, 151, 100, 152, 12, 19, 61, 1, 134, 235, 26, 1, 188, 110, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }, to: Address { payload: BLS([255, 93, 70, 91, 196, 75, 1, 123, 239, 41, 150, 75, 192, 188, 108, 200, 75, 114, 7, 3, 60, 3, 40, 94, 24, 114, 173, 0, 107, 61, 163, 141, 163, 146, 35, 255, 114, 179, 247, 73, 2, 22, 142, 215, 72, 166, 7, 130]) }, sequence: 10015494551597529448, value: TokenAmount(17.944987747335534201), method_num: 11691314341994836873, params: RawBytes { ac8ff874 }, gas_limit: 11122191285217787318, gas_fee_cap: TokenAmount(31757585514871112913072579344521071329437603387853566530475.694921303272763956), gas_premium: TokenAmount(855557112755549049555464473455842432394.361919810497162303) }